src/panels/config/zha/zha-group-page.ts

```diff
@@ -0,0 +1,143 @@
+import {
+  property,
+  LitElement,
+  html,
+  customElement,
+  css,
+  CSSResult,
+  PropertyValues,
+} from "lit-element";
+
+import memoizeOne from "memoize-one";
+
+import "../../../layouts/hass-subpage";
+import "../../../layouts/hass-error-screen";
+import "../ha-config-section";
+import { HomeAssistant } from "../../../types";
+import {
+  ZHADevice,
+  ZHAGroup,
+  fetchGroup,
+  removeGroups,
+} from "../../../data/zha";
+import { formatAsPaddedHex } from "./functions";
+import "./zha-device-card";
+import { navigate } from "../../../common/navigate";
+import "@polymer/paper-icon-button/paper-icon-button";
+
+@customElement("zha-group-page")
+export class ZHAGroupPage extends LitElement {
+  @property() public hass!: HomeAssistant;
+  @property() public group?: ZHAGroup;
+  @property() public groupId!: number;
+  @property() public narrow!: boolean;
+  private _firstUpdatedCalled: boolean = false;
+
+  private _members = memoizeOne(
+    (group: ZHAGroup): ZHADevice[] => group.members
+  );
+
+  public connectedCallback(): void {
+    super.connectedCallback();
+    if (this.hass && this._firstUpdatedCalled) {
+      this._fetchData();
+    }
+  }
+
+  protected firstUpdated(changedProperties: PropertyValues): void {
+    super.firstUpdated(changedProperties);
+    if (this.hass) {
+      this._fetchData();
+    }
+    this._firstUpdatedCalled = true;
+  }
+
+  protected render() {
+    if (!this.group) {
+      return html`
+        <hass-error-screen
+          error="${this.hass.localize(
+            "ui.panel.config.zha.groups.group_not_found"
+          )}"
+        ></hass-error-screen>
+      `;
+    }
+
+    const members = this._members(this.group);
+
+    return html`
+      <hass-subpage .header=${this.group.name}>
+        <paper-icon-button
+          slot="toolbar-icon"
+          icon="hass:delete"
+          @click=${this._deleteGroup}
+        ></paper-icon-button>
+        <ha-config-section .isWide=${!this.narrow}>
+          <div class="header">
+            ${this.hass.localize("ui.panel.config.zha.groups.group_info")}
+          </div>
+          <span slot="introduction">
+            ${this.hass.localize("ui.panel.config.zha.groups.group_details")}
+          </span>
+          <span> <b>Name:</b> ${this.group.name} </span>
```

Comment:
Why is this in a `span`?

Reply:
What should it be instead?

Reply:
You can just remove the span it is nit needed.

Reply:
guessing <p> since no styles are applied?

Reply:
`<p>` is a possibility, but will give you margins and linebreaks, the span doesn't do anything now. So if you want the same behaviour, drop the spans.